vendor/github.com/google/cel-go/parser/gen/cel_parser.go

```diff
@@ -1,10 +1,10 @@
-// Code generated from /usr/local/google/home/tswadell/go/src/github.com/google/cel-go/parser/gen/CEL.g4 by ANTLR 4.13.1. DO NOT EDIT.
+// Code generated from /usr/local/google/home/jdtatum/github/cel-go/parser/gen/CEL.g4 by ANTLR 4.13.1. DO NOT EDIT.
```

Comment:
not-blocking but ... we should probably suggest to upstream that they don't include absolute paths here with home directory names etc? /shrug

Reply:
(this exists in a few files)